src/bootman/update.c

```diff
@@ -35,118 +34,20 @@ bool boot_manager_update(BootManager *self)
         autofree(char) *boot_dir = NULL;
         int did_mount = -1;
 
-        /* Image mode is very simple, no prep/cleanup */
-        if (boot_manager_is_image_mode(self)) {
-                LOG_DEBUG("Skipping to image-update");
-                return boot_manager_update_image(self);
-        }
-
         did_mount = detect_and_mount_boot(self, &boot_dir);
-        if (did_mount >= 0) {
-                /* Do a native update */
-                ret = boot_manager_update_native(self);
-                if (did_mount > 0) {
-                        umount_boot(boot_dir);
-                }
+        ret = boot_manager_update_target(self);
```

Comment:
I am just a little paranoid about what we do when running cbm on a non-root prefix in various cases now. Testing for installing using clr-installer, running with a path and running with a path that is / are all things I'd like to know are working for both systemd and extlinux cases.

Reply:
(and where boot is mounted and not mounted)